solutions/community-deploy/build-cortx.sh

```diff
@@ -129,8 +130,9 @@ function nginx_validation() {
 nginx_validation
 # clone cortx-re repository & run build.sh
 if [ -d $LOCAL_MOUNT/cortx-re ]; then rm -rf $LOCAL_MOUNT/cortx-re; fi
-git clone https://github.com/Seagate/cortx-re -b $BRANCH && pushd cortx-re/docker/cortx-deploy/
-./build.sh -b http://$IP -o rockylinux-8.4 -s all -e opensource-ci
+#git clone https://github.com/Seagate/cortx-re -b $BRANCH && pushd cortx-re/docker/cortx-deploy/
+git clone https://github.com/shailesh-vaidya/cortx-re -b community-deploy && pushd cortx-re/docker/cortx-deploy/
+./build.sh -b http://$IP -o rockylinux-8.4 -s all -e opensource-ci -r local -p yes
```

Comment:
need to add variable for registry option?

Reply:
It's just a `local` string and then the script will set up a local registry.